cmd/ontap-cvo.go

```diff
@@ -513,6 +519,65 @@ func getSvmInfoList(client *kubernetes.Clientset) map[string]SvmInfo {
 	return filerList
 }
 
+func createErrorUserConfigMap(client *kubernetes.Clientset, namespace string, share string, error error) {
+	// Logs the error message for the pod logs
+	klog.Errorf("Error occured for ns %s: %s", namespace, error.Error())
+
+	errorCM, err := client.CoreV1().ConfigMaps(namespace).Get(context.Background(), "shares-error", metav1.GetOptions{})
+	if k8serrors.IsNotFound(err) {
+		//If the error CM doesn't exist, we create it
+		errorData := []ShareError{{
+			Share: share,
+			Error: error.Error(),
+		}}
+		newErrors, err := json.Marshal(errorData)
+		if err != nil {
+			klog.Errorf("Error while mashalling error configmap for %s", namespace)
+		}
+
+		errorCM := corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "shares-error",
+				Namespace: namespace,
+			},
+			Data: map[string]string{"errors": string(newErrors)},
+		}
+		_, err = client.CoreV1().ConfigMaps(namespace).Create(context.Background(), &errorCM, metav1.CreateOptions{})
+		if err != nil {
+			klog.Errorf("Error while creating error configmap for %s", namespace)
+		}
+
+		return
+	} else if err != nil {
+		klog.Errorf("Error while retrieving error configmap for %s", namespace)
+	}
+	//If the error CM does exist, we update it
+	errorData := []ShareError{}
+	json.Unmarshal([]byte(errorCM.Data["errors"]), &errorData)
+
+	errorData = append(errorData, ShareError{
+		Share: share,
+		Error: error.Error(),
+	})
+
+	newErrors, err := json.Marshal(errorData)
+	if err != nil {
+		klog.Errorf("Error while mashalling error configmap for %s", namespace)
```

Comment:
marshalling, typo

Reply:
also, maybe you should dump this error as well so you can debug? just a suggestion (recurring theme for this fn)

Reply:
Will do!